roles/openldap/defaults/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# defaults file for ansible-openldap
+openldap_admin_password: '{{ vault_rciam_openldap.admin_password }}'
```

Comment:
For the default values it's better to remove references to vault and also to comment out the password so that users are forced to provide a value:
```
#openldap_admin_password: "secret"
```